composer/algorithms/stochastic_depth/sample_stochastic_layers.py

```diff
@@ -8,6 +8,12 @@ def _sample_drop(x: torch.Tensor, sample_drop_rate: float, is_training: bool):
     """Randomly drops samples from the input batch according to the `sample_drop_rate`.
```

Comment:
This file needs a module-level docstring ([example](https://github.com/mosaicml/composer/blob/01f671ed3c5cab2e26ec3e8bd201694331413027/composer/algorithms/colout/colout.py#L3)) and an [`__all__`](https://github.com/mosaicml/composer/blob/01f671ed3c5cab2e26ec3e8bd201694331413027/composer/algorithms/colout/colout.py#L28).

Reply:
Ah gotcha, I just added one.